etc/graphql/di.xml

```diff
@@ -1,17 +0,0 @@
-<?xml version="1.0"?>
```

Comment:
@enishant , why we are deleting this one?

Reply:
Included file(s) which was get deleted while git merge ours.